### PR TITLE
Initial service.

### PR DESCRIPTION
**Pull request by austinkelleher**
This initial PR exposes an HTTP endpoint that will serve always serve a "Windbreaker Enabled" SVG badge.

`http://localhost:8080/:owner/:repo,svg` => `http://localhost:8080/windbreaker-io/windbreaker-badges.svg`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "windbreaker-hooks",
```
Comment (Druotic) [inline package.json]: ORLY?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+services:
+- docker
+script: docker-compose run test
+notifications:
+  slack:
+    on_pull_requests: false
+    secure: MFbeaRP3DIUiUtPxmPR/jAofx5XdLxs1OnNBiu48u+i/G0jsLD69p9UCuQoo1Z8y83hS8+rXK/khi00IDK3yZlpfpv8RwaQ5O3bcIw7yie0K6LT7Dy0eGMHqXwM/bivPMfqzhlHOaN190iWRCZvCmGnRiJW8jpSowxjTK15VlBfiyfEpsOfNclpZ1D55AJ7xROQNGG3q2gyxlw4ZwW2VTssfwAvw86iON8+7EO4YcoufaYo7Mn9MgxrYtowW0PXNbwMU1BlTuGn3fKgA3getqrJiFPt8laeuhpjzOYJ2xNQSMKJzlqUWwc41ey0jt9m1fx/I1zsrMEvx5Kc3WRs29LSWjhLwDodJ1MuVxZp8BmoTW6IWPJFTv1cLKcp0GiFyS9RGO529khVDThYAyAvnJJt5uQnC4Nij3/4UdmOCafGlx1S6sas4mJAMhG4O5nGTO6TQuTeNMarSvi2DKjNmcr/t6vSmklaUH3dfy6fgq2Vb/XH/FnVGl8vhiIYWKU+KzJe2txpn4bNoLAdicZZPbO1N0c1HWvljsjjkuGRwtnC8Qwdal3ni/FkQ+G/n/VUsUkYr10RSOUqg9v3wh1ItxseXsxQUdxF+vd8XF8V5818niJN0mQ42cJKtSOawnochakn8HKnUayQ9bRVa9TDZr5f19yna6IG8Cs4QLgCffGk=
```
Comment (Druotic) [inline .travis.yml]: Isn't this just copied from hooks?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM node:8.4.0-alpine
+
+ENV HOME=/usr/windbreaker-hooks
```
Comment (Druotic) [inline Dockerfile]: hooks

File: config/docker.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    port: 7000
+  - host: '127.0.0.1'
+    port: 7000
+knex:
+  client: 'pg'
+  connection:
+    host: 'postgres'
+    user: 'postgres'
+    password: 'postgres'
+    database: 'windbreaker'
```
Comment (Druotic) [inline config/docker.yml]: windbreaker-badges

File: config/localhost.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    port: 7000
+  - host: '127.0.0.1'
+    port: 7000
+knex:
+  client: 'pg'
+  connection:
+    host: 'postgres'
+    user: 'postgres'
+    password: 'postgres'
+    database: 'windbreaker'
```
Comment (Druotic) [inline config/localhost.yml]: windbreaker-badges

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+  windbreaker-hooks:
+    build: .
+    command: 'npm start'
+    depends_on:
+      - postgres
+    volumes:
+      - .:/usr/windbreaker-hooks
```
Comment (Druotic) [inline docker-compose.yml]: hooks?

File: docker-compose.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      - /usr/windbreaker-hooks/node_modules
+    environment:
+      SERVICE_ENVIRONMENT: 'docker'
+  test:
+    build: .
+    command: 'npm run test:docker:ci'
+    depends_on:
+      - postgres
+    volumes:
+      - .:/usr/windbreaker-hooks
```
Comment (Druotic) [inline docker-compose.yml]: hooks?

File: docker-compose.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      - /usr/windbreaker-hooks/node_modules
```
Comment (Druotic) [inline docker-compose.yml]: hooks?

File: docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    environment:
+      SERVICE_ENVIRONMENT: 'docker'
+  postgres:
+    image: 'postgres:9.6.3-alpine'
+    environment:
+      POSTGRES_USER: 'postgres'
+      POSTGRES_PASSWORD: 'postgres'
+      POSTGRES_DB: 'windbreaker'
```
Comment (Druotic) [inline docker-compose.yml]: windbreaker-badges

File: package.json [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+  "devDependencies": {
+    "ava": "^0.20.0",
+    "browser-refresh": "^1.6.0",
+    "eslint": "^4.4.1",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-import": "^2.3.0",
+    "eslint-plugin-node": "^5.1.1",
+    "eslint-plugin-promise": "^3.5.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "get-port-sync": "^1.0.0",
```
Comment (Druotic) [inline package.json:55]: Are all of these dependencies used?

File: src/startup-tasks/index.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
+function _createTasks () {
+  let tasks = require('./tasks')
+
+  for (let i = 0; i < tasks.length; i++) {
+    let taskName = tasks[i]
+    let task = tasks[i] = require('./' + taskName)
+    task.name = taskName
+  }
+
+  return tasks
```
Comment (Druotic) [inline src/startup-tasks/index.js:10]: Optional, but `.map` could be used here to simply this a bit.

**Review by Druotic — CHANGES_REQUESTED** [review]

**Comment by austinkelleher** [comment]
[Windbreaker Badges Service](https://trello.com/c/72Fie46a/10-windbreaker-badges-service)

**Pull request merged**